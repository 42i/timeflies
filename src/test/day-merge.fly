wp a
wp b

day 2012-10-15 8 16
- a 5

day 2012-10-16
- a 2.5

day 2012-10-15
- b 3

day 2012-10-16 9 15
- b 3.5

day 2012-10-17 7 11

day 2012-10-17 7 11