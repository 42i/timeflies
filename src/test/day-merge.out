day-merge.fly:18: ERROR : day 2012-10-17 redefined.
Day check (all):
2012-10-17 Wed: worked  4.00, allocated  0.00, delta -4.00
1 problem detected.
Time at work overview (all):
     when        worked   leave    sick balance
2012-10-15 Mon:    8.00 ----.-- ----.-- ----.--
2012-10-16 Tue:    6.00 ----.-- ----.--   -2.00
2012-10-17 Wed:    4.00 ----.-- ----.--   -4.00
  week 2012-42:   18.00 ----.-- ----.--   -6.00
 month 2012-10:   18.00 ----.-- ----.--   -6.00
         total:   18.00 ----.-- ----.--   -6.00
     when        worked   leave    sick balance
Work package summary (all):
  14.00 : ALL
       7.50 : a
              - 2012-10-15 5.0
              - 2012-10-16 2.5
       6.50 : b
              - 2012-10-15 3.0
              - 2012-10-16 3.5
